### PR TITLE
DM-43090: next-visit-fan-out for LSSTComCamSim for dev

### PR DESCRIPTION
**Pull request by hsinfang**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -159,6 +159,7 @@ async def main() -> None:
     offset = os.environ["OFFSET"]
     kafka_schema_registry_url = os.environ["KAFKA_SCHEMA_REGISTRY_URL"]
     latiss_knative_serving_url = os.environ["LATISS_KNATIVE_SERVING_URL"]
```
Comment (hsinfang) [inline src/main.py:161]: Spotted my typo in the commit message (LSSTComCam not LSSTCamCam); will fix that.

File: src/main.py [hunk 2]
```diff
@@ -305,10 +315,17 @@ async def main() -> None:
                             )
                             knative_serving_url = latiss_knative_serving_url
                             in_process_requests_gauge = latiss_in_process_requests_gauge
-                        # case "LSSTComCam":
-                        #    fan_out_message_list = next_visit_message.add_detectors(
-                        #        next_visit_message, lsst_com_cam_active_detectors
-                        #    )
+                        case 3:  # LSSTComCamSim
```
Comment (kfindeisen) [inline src/main.py:319]: I assume that if the Summit scheduler does something different, we'll adjust the dev system to match?

File: src/main.py [hunk 3]
```diff
@@ -159,6 +159,7 @@ async def main() -> None:
     offset = os.environ["OFFSET"]
     kafka_schema_registry_url = os.environ["KAFKA_SCHEMA_REGISTRY_URL"]
     latiss_knative_serving_url = os.environ["LATISS_KNATIVE_SERVING_URL"]
+    lsstcomcam_knative_serving_url = os.environ["LSSTCOMCAM_KNATIVE_SERVING_URL"]
```
Comment (kfindeisen) [inline src/main.py:162]: I guess we already have a URL defined in the Phalanx config, but this makes me a bit nervous -- trying to support all the instruments in advance has caused us a lot of unnecessary maintenance...

**Review by kfindeisen — APPROVED** [review]
I'm a bit nervous about supporting the real ComCam in advance, otherwise looks good.

File: src/main.py (hunk 2 above)
Comment (hsinfang) [reply src/main.py:319]: Yes, that will be [DM-42713](https://jira.lsstcorp.org/browse/DM-42713).

**Comment by hsinfang** [comment]
My thinking on adding the ComCam part is that the LSSTCam counterpart already exists but ComCam partially exists. I would prefer either they all exist or they all don't exist.  So instead of removing the existing pieces, I opted for completing (to my best effort) the ComCam counterpart.

**Pull request merged**